### ISSUE TITLE
Redirect link to expired link scan failure

### ISSUE THREAD

**Issue by MattariOnline** [ordubis/RedditDSBot#6]
Redirect link to expired link scan failure

Output:
Handling submission 9hcbs1 by KarmaEverywhere
  Link: http://discord.plus/test
http://discord.plus/test uses 302 to redirect to https://discord.plus/
  After following redirects found final url https://discord.plus/
  Ignoring; the final url was not an official link
Sleeping for 5 seconds

Notes:
This should remove the post as I specifically created a redirect link that didn't have an actual server link to forward to, but rather, a fake server link. Likely same post-removal issue as mentioned in first test failure.

**Comment by Tjstretchalot** [comment]
Format as a log by surrounding it with three backticks, ex:

```
Output:
Handling submission 9hcbs1 by KarmaEverywhere
Link: http://discord.plus/test
http://discord.plus/test uses 302 to redirect to https://discord.plus/
After following redirects found final url https://discord.plus/
Ignoring; the final url was not an official link
Sleeping for 5 seconds
```

**Comment by Tjstretchalot** [comment]
This was fixed with #7 I suggest closing
